src/extension/src/coreTemplateStudio.ts

```diff
@@ -40,7 +38,7 @@ export class CoreTemplateStudio {
     }
 
     let platform = process.platform;
```

Comment:
`const` here since platform is not being modified after declaration

Reply:
updated in my PR :)